MantidQt/MantidWidgets/src/DataProcessorUI/DataProcessorOneLevelTreeManager.cpp

```diff
@@ -388,10 +388,27 @@ void DataProcessorOneLevelTreeManager::update(
                      QString::fromStdString(data[col]));
 }
 
+/** Sets a new row to be highlighted
+* @param position : The index of the row to be highlighted
+*/
+void DataProcessorOneLevelTreeManager::addHighlighted(int position) {
+  m_model->addHighlighted(position);
+}
+
+/** Sets a new row to be highlighted
+* @param position : The index of the row to be highlighted
+* @param parent : The parent of the row
+*/
```

Comment:
There are two versions of the method `addHighlighted` that are required to be implemented as they are inherited from `DataProcessorTreeManager` which defines them as virtual methods. Despite that it doesn't need the parent, it could still be expected to receive one.